estate/models/estate_property.py

```diff
@@ -0,0 +1,45 @@
+from odoo import fields, models
+from dateutil.relativedelta import relativedelta
+
+
+class Property(models.Model):
```

Comment:
```suggestion
class EstateProperty(models.Model):
```
It won't have big incidence here but we generally put the name of the file here, in CamelCase.

Reply:
This is now updated. I just want to check if we have a convention for this. As in our [Chapter 3](https://www.odoo.com/documentation/19.0/developer/tutorials/server_framework_101/03_basicmodel.html) example [CRM Recurring Plan](https://github.com/odoo/odoo/blob/e80911aaead031e7523173789e946ac1fd27c7dc/addons/crm/models/crm_recurring_plan.py#L1-L9), we omit the add on name and just use the model for class name, so I was inferring that we have a specific naming convention, like add-on-name_model-name for file, and model-name only for class name.

Reply:
No it's not a strong official convention, more an employee tacit thing, and it's indeed not always the case.
Always remember that some part of the Odoo code are 10 years old, when there was 20 devs :wink: